### PR TITLE
[CHEAP-662 + CHEAP-666 + CHEAP-667] - Store UI, Colour Palette Section, Image Uploader

### PR DESCRIPTION
**Pull request by fpunny**
Rip anyone doing this PR

### Relevant changes
 - /components/container/ImageCard
 - /components/input/Image
 - /stories/Colors

### TODO
 - Add more magic to image upload
 - Get a color parser for color palette

All is avaliable on the demo site
https://react-ui.cheapreats.com

### CODE REVIEW

File: src/components/buttons/Button.js [hunk 1]
```diff
@@ -1,98 +1,127 @@
...
+    border-radius: 999px;
+    font-weight: bold;
+    border: none;
+    outline: none;
+    cursor: pointer;
+    
+    ${({ margin }) => `margin: ${ margin + (typeof(margin) === 'string' ? '' : 'px') };`}
+    ${({ disabled }) => !disabled ? css`
+        &:hover, &:focus {
+            background-color: ${props => props.primary ? "#B22330" : "#f4f4f4"};
```
Comment (ralph-dev) [inline src/components/buttons/Button.js]: WHAT ARE THESE MAGIC VARIABLES

File: src/components/inputs/Image.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+`;
+
+const CropWrapper = styled.div`
+    ${ flex('row', 'center') }
+    background-color: ${ ({ theme }) => theme.colors.inputBackground.default };
+    padding: 20px;
+`;
+
+export const Image = ({ accept = 'image/*', onChange = () => {}, ...props }) => {
+    const [ crop, setCrop ] = useState({ aspect: 2/1 });
```
Comment (ralph-dev) [inline src/components/inputs/Image.js]: magic

Comment (fpunny) [reply src/components/inputs/Image.js]: Ya the stuff i need to touch up on :p

File: src/components/buttons/Button.js (hunk 1 above)
Comment (fpunny) [reply src/components/buttons/Button.js]: D: i found them like that

**Pull request merged**